flashcards/backend/decks/admin.py

```diff
@@ -15,4 +15,17 @@ class CategoryAdminConfig(admin.ModelAdmin):
     list_per_page = 50
 
 
+class DifficultyLevelAdminConfig(admin.ModelAdmin):
+    """Class **DifficultyAdminConfig**
+    displays difficulty levels list in admin panel based on :class:`decks.models.DifficultyLevel` model."""
+
+    list_display = ("name", "value")
+    search_fields = ("name",)
+    list_editable = ("value",)
+    list_display_links = ("name",)
+    list_filter = ("name",)
```

Comment:
We have `unique=True` on `DifficultyLevel` model. Filtering by name will result in listing all names on the right side of the admin panel. Instead I would add a default ordering by name.